helm/chart/maesh/charts/metrics/values.yaml

```
@@ -4,6 +4,15 @@ image:
   configmapReload: jimmidyson/configmap-reload:v0.2.2
 
 grafana:
+  service:
+    type: ClusterIP
+    port: 3000
+
+    ## Specify the nodePort value for the NodePort service types.
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#nodeport
+    ##
+    # nodePort: null
```

I think that it's better to use a dummy `nodePort` for the example instead of `null`.

```suggestion
    # nodePort: 30030
```